app/models/user.rb

```diff
@@ -5,4 +5,9 @@ class User < ApplicationRecord
   has_many :books, through: :checkouts
 
   validates :name, presence: true
+
+  def total_fines
+    total = checkouts.inject(0) { |total, checkout| total + checkout.fine }
+    total = sprintf('%.2f', total)
```

Comment:
@derekprior, Oddly, it does not work if I remove that formatting. I have been unable to figure out why :(